client/allocrunner/taskrunner/envoybootstrap_hook.go

```diff
@@ -18,6 +18,8 @@ import (
 
 var _ interfaces.TaskPrestartHook = &envoyBootstrapHook{}
 
+const envoyBaseAdminPort = 19000
```

Comment:
will this be exposed as an env var? we would want to scrape data from envoy admin port for all our allocs, and need to dynamically discover these endpoints